### PR TITLE
Js my goals logic

### PR DESCRIPTION
**Pull request by JustynaSmoliga**
Żeby uruchomić unitimplementation należy wpisać npm run start:myGoals.

### CODE REVIEW

**Review by skorpiom — APPROVED** [review]
Hey,
Components look and work great!
I left just two suggestions.

**Review by robert-czwartosz — COMMENTED** [review]
Component works great!
There is some issue with unit tests.
 FAIL  Test/HistoricalWeeklyGoalsTable.test.ts
  ● Tests for createHistoricalWeeklyGoalsTable › Should render table with rows
expect(received).toMatchSnapshot()

    Snapshot name: `Tests for createHistoricalWeeklyGoalsTable Should render table with rows 1`
snapshot --->  17/12/1995
received --->  12/17/1995

snapshot ---> 18/01/1995
received ----> 1/18/1995

**Comment by JustynaSmoliga** [comment]
> Component works great!
> There is some issue with unit tests.
> FAIL Test/HistoricalWeeklyGoalsTable.test.ts
> ● Tests for createHistoricalWeeklyGoalsTable › Should render table with rows
> expect(received).toMatchSnapshot()
> 
> ```
> Snapshot name: `Tests for createHistoricalWeeklyGoalsTable Should render table with rows 1`
> ```
> 
> snapshot ---> 17/12/1995
> received ---> 12/17/1995
> 
> snapshot ---> 18/01/1995
> received ----> 1/18/1995

Do you have latest codes? I have done build on github again, restarted tests and all passed. I can't see any error.

File: Src/UIComponents/HistoricalWeeklyGoalsTable/HistoricalWeeklyGoalsTable.ts [hunk 1]
```diff
@@ -1,35 +1,40 @@
...
-    const title = document.createElement('p');
-    title.className = 'tile-title';
-    const titleContent = document.createTextNode('Historical weekly goals');
-    title.appendChild(titleContent);
-    const tile=generateTileComponent(container);
+function createHistoricalWeeklyGoalsTable(tableData: Goal[]): HTMLDivElement {
+  const container = createElement('div','historical-weekly-goals-table');
+  container.classList.add('tile-container');
+  const title = document.createElement('p');
+  title.className = 'tile-title';
```
[the remark this answers is not in the document]
Comment (JustynaSmoliga) [reply Src/UIComponents/HistoricalWeeklyGoalsTable/HistoricalWeeklyGoalsTable.ts]: done

File: Src/UIComponents/HistoricalWeeklyGoalsTable/HistoricalWeeklyGoalsTable.ts [hunk 2]
```diff
@@ -1,35 +1,40 @@
...
+  const tile = generateTileComponent(container);
 
-    const table=createTable(['DATE','WEEKLY GOAL','START WEIGHT','END WEIGHT','ACHIEVED IN']);
-    const addRowToHistoricalWeeklyGoalsTable = addRow(table);
+  const table = createTable([
+    'DATE',
+    'WEEKLY GOAL',
+    'START WEIGHT',
+    'END WEIGHT',
+    'ACHIEVED IN',
```
[the remark this answers is not in the document]
Comment (JustynaSmoliga) [reply Src/UIComponents/HistoricalWeeklyGoalsTable/HistoricalWeeklyGoalsTable.ts]: done

**Review by robert-czwartosz — APPROVED** [review]

**Pull request merged**